pkg/config/config.go

```diff
@@ -47,16 +47,31 @@ var c config
 var (
 	EnabledEBPFCgroupID = false
 
-	EstimatorModel        = "" // auto-select
-	EstimatorSelectFilter = "" // no filter
-	CoreUsageMetric       = "curr_cpu_cycles"
-	DRAMUsageMetric       = "curr_cache_miss"
-	UncoreUsageMetric     = ""                // no metric (evenly divided)
-	GeneralUsageMetric    = "curr_cpu_cycles" // for uncategorized energy; pkg - core - uncore
+	EstimatorModel        = getConfig("EstimatorModel", "")        // auto-select
+	EstimatorSelectFilter = getConfig("EstimatorSelectFilter", "") // no filter
+	CoreUsageMetric       = getConfig("CoreUsageMetric", "cpu_instr")
+	DRAMUsageMetric       = getConfig("DRAMUsageMetric", "cache_miss")
+	UncoreUsageMetric     = getConfig("UncoreUsageMetric", "")           // no metric (evenly divided)
+	GeneralUsageMetric    = getConfig("GeneralUsageMetric", "cpu_instr") // for uncategorized energy; pkg - core - uncore
 
 	versionRegex = regexp.MustCompile(`^(\d+)\.(\d+).`)
+
+	ConfigPath = "/etc/config"
 )
 
+func getConfig(key, defaultValue string) string {
+	configFile := fmt.Sprintf("%s/%s", ConfigPath, key)
```

Comment:
@sunya-ch @marceloamaral if I read this correctly, we are actually expecting multiple files for multiple metrics
is this work as design? should we create a config file and read KV from it ? I think it's more scale?

Reply:
Good point! I think it should be only one config file

Reply:
When we set the ConfigMap CR of Kubernetes, 

```yaml
data:
  key1: val1
  key2: val2
```
It will automatically create two files.
<mount_path>/key1, <mount_path>/key2
Content is the value for each key.

If we want them to be in the a single file, we have to create CR like this
```yaml
data: |
   key1=var1
   key2=var2
```
It will create single file with the below content.
```
key1=var1
key2=var2
```

The first type is configmap by design of kubernetes. We can also map only subset of configuration values to the pod by key specification, for example, in case the case that we have a shared configuration between model-server and kepler.